example/mini_core.rs

```diff
@@ -563,11 +564,6 @@ unsafe fn allocate(size: usize, _align: usize) -> *mut u8 {
     libc::malloc(size)
 }
 
-#[lang = "box_free"]
-unsafe fn box_free<T: ?Sized>(ptr: Unique<T>, _alloc: ()) {
```

Comment:
Shall stale comments like [this](https://github.com/rust-lang/rust/blob/098d4fd74c078b12bfc2e9438a2a04bc18b393bc/library/alloc/src/boxed.rs#L195) be updated after this removal?

Reply:
Should probably have been done in https://github.com/rust-lang/rust/pull/100036 which did the actual removal of `box_free`, but yes it should be updated.

Reply:
Opened https://github.com/rust-lang/rust/pull/120372